sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -234,6 +233,7 @@ export default {
           );
           break;
       }
+      loading.value = false;
```

Comment:
Just to check, are we planning to use loader for all our data-tables?

Reply:
If it that is a PrimeVue specific feature I would avoid spending time on it.

Reply:
`loading` is primevue dataTable property